extras/rya.export/export.api/src/main/java/org/apache/rya/export/api/parent/MergeParentMetadata.java

```diff
@@ -36,9 +49,33 @@ public Date getTimestamp() {
         return timestamp;
     }
 
+    /**
+     * @return - The copy tool split time metadata for the table.
+     */
+    public Date getCopyToolInputTime() {
```

Comment:
Copy tool input time is the time filter that the user input when running the copy tool.  Copy tool run time is the time when the copy tool ran.  Parent time offset is the time difference between the parent machine's local time and the NTP server time which is used for time synchronization when merging.

We discussed this and one of these fields is a duplicate of "timestamp" and the other field is being renamed "filterTimestamp".